src/api.v2/model/Sample.js

```diff
@@ -81,6 +90,71 @@ class Sample extends BasicModel {
       );
     });
   }
+
+  /**
+   * Creates copy samples from one experiment to another one
+   *
+   * @param {*} fromExperimentId
+   * @param {*} toExperimentId
+   */
+  async copyTo(fromExperimentId, toExperimentId, samplesOrder) {
+    const result = await this.getSamples(fromExperimentId);
```

Comment:
Maybe call this var samples or fromSamples?

Reply:
Renamed to `fromSamples`